nuxeo-core/nuxeo-core-test/src/test/java/org/nuxeo/ecm/core/bulk/DocumentSetRepositoryInit.java

```diff
@@ -36,6 +41,16 @@
     public void populate(CoreSession session) {
         super.populate(session);
         DocumentModel test = session.getDocument(new PathRef("/default-domain/workspaces/test"));
+
+        ACP acp = new ACPImpl();
+        ACL acl = new ACLImpl();
+        acl.add(new ACE("Administrator", "Everything", true));
+        acl.add(new ACE("Tutu", "WriteProperties", true));
```

Comment:
interesting case what will happen for a user that has read access on the entire repository and write access only to few documents, some action could be reserved to admin only.